coveralls-maven-plugin fails on Java 9

Build fails with the following error running under Java 9, probably something to do with their new package restrictions:
```
[ERROR] Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.3.0:report
 (default-cli) on project HikariCP: Execution default-cli of goal org.eluder.coveralls:coveralls-maven-
plugin:4.3.0:report failed: A required class was missing while executing 
org.eluder.coveralls:coveralls-maven-plugin:4.3.0:report: javax/xml/bind/DatatypeConverter
```

--- a/pom.xml
+++ b/pom.xml
@@ -64,6 +64,11 @@
       <artifactId>httpmime</artifactId>
     </dependency>
     <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.10</version>
+    </dependency>
+    <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
     </dependency>
@@ -124,7 +129,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <version>3.4</version>
+      <version>3.6</version>
       <scope>test</scope>
     </dependency>
     <dependency>

--- a/src/main/java/org/eluder/coveralls/maven/plugin/util/Md5DigestInputStream.java
+++ b/src/main/java/org/eluder/coveralls/maven/plugin/util/Md5DigestInputStream.java
@@ -26,11 +26,11 @@ package org.eluder.coveralls.maven.plugin.util;
  * %[license]
  */
 
-import javax.xml.bind.DatatypeConverter;
 import java.io.InputStream;
 import java.security.DigestInputStream;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import org.apache.commons.codec.binary.Hex;
 
 public class Md5DigestInputStream extends DigestInputStream {
 
@@ -39,6 +39,6 @@ public class Md5DigestInputStream extends DigestInputStream {
     }
 
     public String getDigestHex() {
-        return DatatypeConverter.printHexBinary(getMessageDigest().digest());
+        return new String(Hex.encodeHex(getMessageDigest().digest(),false));
     }
 }

--- a/src/test/java/org/eluder/coveralls/maven/plugin/util/TestIoUtil.java
+++ b/src/test/java/org/eluder/coveralls/maven/plugin/util/TestIoUtil.java
@@ -26,9 +26,6 @@ package org.eluder.coveralls.maven.plugin.util;
  * %[license]
  */
 
-import org.codehaus.plexus.util.IOUtil;
-
-import javax.xml.bind.DatatypeConverter;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
@@ -38,8 +35,9 @@ import java.io.PrintWriter;
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.nio.charset.Charset;
-import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.codehaus.plexus.util.IOUtil;
 
 public class TestIoUtil {
 
@@ -77,9 +75,7 @@ public class TestIoUtil {
     }
 
     public static String getMd5DigestHex(final String content) throws NoSuchAlgorithmException {
-        MessageDigest md = MessageDigest.getInstance("MD5");
-        byte[] digest = md.digest(content.getBytes());
-        return DatatypeConverter.printHexBinary(digest);
+        return DigestUtils.md5Hex(content).toUpperCase();
     }
 
     private static URL getResourceUrl(final String resource) {

